Can't change int -> var because:
```
OpenRA.Game/Input/Hotkey.cs(32,5): error CS0818: An implicitly typed local variable declarator must include an initializer
OpenRA.Game/Input/Hotkey.cs(33,37): error CS0841: A local variable `c' cannot be used before it is declared
OpenRA.Game/Input/Hotkey.cs(35,20): error CS0841: A local variable `c' cannot be used before it is declared
```

--- a/OpenRA.Game/Input/Hotkey.cs
+++ b/OpenRA.Game/Input/Hotkey.cs
@@ -28,7 +28,12 @@ namespace OpenRA
 
 			Keycode key;
 			if (!Enum<Keycode>.TryParse(parts[0], true, out key))
-				return false;
+			{
+				int c;
+				if (!int.TryParse(parts[0], out c))
+					return false;
+				key = (Keycode)c;
+			}
 
 			var mods = Modifiers.None;
 			if (parts.Length >= 2)

